2-mandatory/4-restaurant.js

```diff
@@ -20,8 +20,27 @@ const MENU = {
 };
 
 let cashRegister = {
+  orderBurger: (balance) => {
+    let yourPacket = 0;
+    if (balance >= MENU.burger) {
+      yourPacket = balance - MENU.burger;
+    } else {
+      yourPacket = balance;
+    }
+    return yourPacket;
+  },
+  orderFalafel: (balance) => {
+    let yourPacket = 0;
+    if (balance >= MENU.falafel) {
+      yourPacket = balance - MENU.falafel;
+    } else {
+      yourPacket = balance;
+    }
+    return yourPacket;
+  },
```

Comment:
Good, this works well. You could set yourPacket to equal balance at the start of the method and then you don't need the else clause, but either way is good.